### PR TITLE
Home-open event updates and links across events and multitab pages

### PR DESCRIPTION
**Pull request by TimothyKopec**
https://jira.hl7.org/browse/FHIR-41178

### CODE REVIEW

File: input/pagecontent/4-4-multitab-considerations.md [hunk 1]
```diff
@@ -51,8 +51,6 @@ Often an application knowing that it is being driven by an external actor remove
 
 ### Launching A Context-Less Tab
 
-Many applications can have a "home" or "default" tab that contains no clinical context, but may hold useful application features. In some cases other applications may want to subscribe to and be notified when another application has switched to the no context tab. To avoid confusion with other events, a new event is proposed to represent a user switching to this context-less tab.
-
-{% include infonote.html text='Implementer feedback is solicited around the semantics of communicating a context change to a "context-less tab". For example, why not a Patient-open (or ImagingStudy-open or ...) with a patient (or study or ...).' %}
+Many applications can have a "home" or "default" tab that contains no clinical context, but may hold useful application features. In some cases other applications may want to subscribe to and be notified when another application has switched to the no context tab.
 
 Since we are inherently representing the lack of context, the event will not fully conform to the defined event naming syntax and will instead use a static name (similar to `UserLogout.html`).
```
Comment (isaacvetter) [inline input/pagecontent/4-4-multitab-considerations.md:56]: The [Home-open event](3-2-5-Home-open.html) represents a lack of context.

Comment (isaacvetter) [inline input/pagecontent/4-4-multitab-considerations.md:56]:
```suggestion
The [Home-open event](3-2-5-Home-open.html) represents a lack of context.
```

**Review by isaacvetter — CHANGES_REQUESTED** [review]

**Comment by bvdh** [comment]
I've added some text improvements, added a comment that the event does not change current context

File: input/pagecontent/3-2-5-Home-open.md [hunk 2]
```diff
@@ -8,11 +8,11 @@ The user has opened or switched back to the application's home page or tab which
 
 Unlike most of FHIRcast events, `Home-open` is representing the lack of a FHIR resource context and therefore does not fully follow the `FHIR-resource`-`[open|close]` syntax.
 
-The order of patients can be different between different applications (like a late application joining being temporarily out of sync).
+Note that sending a `Home-open` event merely signals that an application has switched to the no-context tab. It does not change the current context. For instance, if a specific patient is currently in context, sending a Home-open event does not imply that this patient is no longer in context.
```
Comment (isaacvetter) [inline input/pagecontent/3-2-5-Home-open.md:11]:
```suggestion
```

File: input/pagecontent/3_Events.md [hunk 3]
```diff
@@ -12,8 +12,9 @@ The sections in this chapter are:
 
 | **3.2 Infrastructure Events** |
 | [3.2.1 SyncError event](3-2-1-SyncError.html) |
-| [3.2.2 UserLogout.html event](3-2-2-UserLogout.html) |
-| [3.2.3 UserHibernate.html event](3-2-3-UserHibernate.html) |
+| [3.2.3 UserLogout.html event](3-2-3-UserLogout.html) |
```
Comment (isaacvetter) [inline input/pagecontent/3_Events.md:15]:
```suggestion
| [3.2.3 UserLogout.html event](3-2-2-UserLogout.html) |
```

File: input/pagecontent/4-4-multitab-considerations.md [hunk 4]
```diff
@@ -51,8 +52,6 @@ Often an application knowing that it is being driven by an external actor remove
...
+Many applications feature a "home" or "default" tab that contains no clinical context but may offer useful application features. Sometimes, other applications may want to subscribe to and be notified when an application switches to this no-context tab. This is indicated by the [Home-open event](3-2-5-Home-open.html).
 
-{% include infonote.html text='Implementer feedback is solicited around the semantics of communicating a context change to a "context-less tab". For example, why not a Patient-open (or ImagingStudy-open or ...) with a patient (or study or ...).' %}
-
-Since we are inherently representing the lack of context, the event will not fully conform to the defined event naming syntax and will instead use a static name (similar to `UserLogout.html`).
+Note that sending a [Home-open event](3-2-5-Home-open.html) merely signals that an application has switched to the no-context tab. It does not change the current context. For instance, if a specific patient is currently in context, sending a Home-open event does not imply that this patient is no longer in context.
```
Comment (isaacvetter) [inline input/pagecontent/4-4-multitab-considerations.md:57]:
```suggestion
```

**Review by isaacvetter — APPROVED** [review]

**Pull request merged**